quiz/templates/quiz/quiz.html

```diff
@@ -0,0 +1,117 @@
+{% extends "_base.html" %}
+
+{% block content %}
+    <!-- general form elements -->
+    <div class="box box-primary">
+    <div class="box-header with-border">
+        <h3 class="box-title">Quiz</h3>
+    </div>
+    <!-- /.box-header -->
+    <!-- form start -->
+    <form role="form" onsubmit="return submitAnswer();">
+        {% csrf_token %}
+        <div class="box-body">
+        <div class="form-group">
+            <label id="question"></label>
+        </div>
+            <div id="answers"> </div> 
+        </div>
+        <!-- /.box-body -->
+        <div class="box-footer">
+            <button class="btn btn-primary">Submit</button>
+        </div>
+    </form>
+    </div>
+    <!-- /.box -->
+{% endblock content %}
+
+{% block javascript  %}
+    {{ block.super }}
```

Comment:
See I like to do this:
```python
{% block javascript %}
    ...
    {% block extra_javascript %}{% endblock %}
{% endblock %}
```

In the parent template. That way you don't have to override and then call super in the page template, you can just go ahead and override the empty `{% block extra_javascript %}`.